bin/lib/cli/environment.py

```diff
@@ -96,6 +102,12 @@ def environment_refresh(cfg: Config, min_healthy_percent: int, motd: str):
                 last_log = log
             if status in ('Successful', 'Failed', 'Cancelled'):
                 break
+    if not no_notify:  # Double negation because I don't know how to make a default cli flag be True
+        current_notify = get_current_notify()
```

Comment:
you should check if cfg.env == prod, so it doesnt notify on beta or staging

Reply:
I _knew_ I was missing something, hah